root/area/edit_form.tt

```diff
@@ -42,6 +42,16 @@
 
 <script type="text/javascript">//<![CDATA[
   (function () {
+    const wasEnded = $('#id-edit-area\\\.period\\\.ended').prop('checked');
```

Comment:
@mwiencek: Thank you for the review. By the way, alias and most of entity types are using the same date range form. Shouldn’t that be factorized as well?